tests/python_on_whales/components/test_image.py

```diff
@@ -16,31 +17,32 @@ def test_load_json(json_file):
     # we could do more checks here if needed
 
 
-def test_image_repr():
-    docker.image.pull("busybox:1", quiet=True)
-    docker.image.pull("busybox:1.32", quiet=True)
-    assert "busybox:1" in repr(docker.image.list())
-    assert "busybox:1.32" in repr(docker.image.list())
-    docker.image.remove(["busybox:1", "busybox:1.32"])
+def test_image_repr(ctr_client: DockerClient):
+    ctr_client.image.pull("busybox:1", quiet=True)
+    ctr_client.image.pull("busybox:1.32", quiet=True)
+    assert "busybox:1" in repr(ctr_client.image.list())
+    assert "busybox:1.32" in repr(ctr_client.image.list())
+    ctr_client.image.remove(["busybox:1", "busybox:1.32"])
 
 
-def test_image_remove():
-    docker.image.pull("busybox:1", quiet=True)
-    docker.image.pull("busybox:1.32", quiet=True)
-    docker.image.remove(["busybox:1", "busybox:1.32"])
+def test_image_remove(ctr_client: DockerClient):
+    ctr_client.image.pull("busybox:1", quiet=True)
+    ctr_client.image.pull("busybox:1.32", quiet=True)
+    ctr_client.image.remove(["busybox:1", "busybox:1.32"])
 
 
-def test_image_save_load(tmp_path):
+@pytest.mark.ctr_mgr("docker")
+def test_image_save_load(ctr_client: DockerClient, tmp_path: Path):
```

Comment:
Having a custom market may lead to some complexity in case someone wants to understand, debug or change the mechanism. I'd be in favor of something more explicit, like 

```python
@pytest.mark.parametrize("ctr_client", [DOCKER_CLIENT, PODMAN_CLIENT])
def test_image_save_load(ctr_client: DockerClient, tmp_path: Path):
    ...
```
It's easy to debug and understand. DOCKER_CLIENT and PODMAN_CLIENT can be defined somewhere else. It could even be defined with the pytest CLI in case someone doesn't have a docker engine locally, and can use the client with ssh for example (but that's not an immediate need).